R/map_autographr.R

```diff
@@ -1039,6 +1003,20 @@ map_dynamic <- function(edges_out, nodes_out, edge_color, node_shape,
   p
 }
 
+check_color <- function(v) {
```

Comment:
New standard suggestion: functions that are only meant to be used internally should be prefixed with a period, e.g. `.check_color`